src/lib/y2users/linux/writer.rb

```diff
@@ -158,6 +165,68 @@ def add_user(user, issues)
         log.error("Error creating user '#{user.name}' - #{e.message}")
       end
 
+      # Command for creating new users
+      GROUPADD = "/usr/sbin/useradd".freeze
```

Comment:
Do you mean [groupadd](https://www.man7.org/linux/man-pages/man8/groupadd.8.html)?

Reply:
good catch